Tests/NIOTests/CodecTest.swift

```diff
@@ -200,22 +200,24 @@ public final class ByteToMessageDecoderTest: XCTestCase {
         XCTAssertEqual(testDecoderIsNotQuadratic_reallocs, 3)
     }
 
-    func testMemoryIsReclaimedIfMostIsConsumed() throws {
+    func testMemoryIsReclaimedIfMostIsConsumed() {
         let channel = EmbeddedChannel()
         defer {
-            XCTAssertNoThrow(try channel.finish())
+            XCTAssertNoThrow(XCTAssertTrue(try channel.finish().isClean))
```

Comment:
I think we can get rid of the enclosing `XCTAssertNoThrow`...

Reply:
@fabianfett but we want that?

Reply:
it has a `try` and I don't want it to throw, right?

Reply:
Correct, but AFAIK XCTAssertTrue also catches `try`.

Reply:
In other word `XCTAssertTrue` has a build in `XCTAssertNoThrow`

Reply:
Probably not available for Swift 5.0.